frontend/src/store/modules/auth/actions.js

```diff
@@ -1,9 +1,24 @@
 import {LOGIN, LOGOUT} from './types/actions';
 import {SET_AUTHENTICATED_USER, USER_LOGIN, USER_LOGOUT} from "./types/mutations";
+import requestService from "../../../services/requestService";
 
 export default {
-    [LOGIN]: (context, user) => {
-        return new Promise((resolve, reject) => {
+    [LOGIN]: async (context, user) => {
+        const url = process.env.VUE_APP_API_URL + '/auth/login';
+
+        try {
+            const response = await requestService.create(url, user);
+
+            context.commit(USER_LOGIN, response);
+            context.commit(SET_AUTHENTICATED_USER, user);
```

Comment:
User should be fetched from `me` endpoint, no need to save password value to state, it's unsecured